### PR TITLE
doc(packages): format docs code

### PR DESCRIPTION
**Pull request by wzc520pyfm**
The current document style is not uniform, I formatted it according to the Basic Components section.

### CODE REVIEW

File: packages/ui/src/SideTip/index.md [hunk 1]
```diff
@@ -78,20 +78,20 @@ nav:
...
+| loading | 是否为加载中状态 | boolean | false | - |
+| position | 初始位置 | { right: number, bottom: number } | {right: 24, bottom: 24} | - |
+| badge | antd Badge 组件相关属性 | 参考 <a href="https://ant.design/components/badge-cn/#API" target="_blank">badge</a> 组件 | - | - |
+| tooltip | antd Tooltip 组件相关属性 | 参考 <a href="https://ant.design/components/tooltip-cn/#API" target="_blank">tooltip</a> 组件 | - | - |
+| id | 若页面中有多个侧边提示组件，则通过 id 标识当前组件，用于缓存侧边提示组件是否隐藏配置 | string | - | - |
+| onDragStart | 开始拖动的方法回调 | () => void | - | - |
+| onDragEnd | 结束拖动的方法回调 | () => void | - | - |
+| onDrag | 拖动中的方法回调 | (offset: {right: number, bottom: number}) => void | - | - |
+| buttonStyle | 按钮的样式 | React.CSSProperties | - | 2.10.3 | - |
+| buttonClassName | 按钮的类 | string | - | 2.10.3 | - |
```
Comment (dengfuping) [inline packages/ui/src/SideTip/index.md:97]: - should remove `2.10.3` version info.

File: packages/charts/src/DualAxes/index.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ nav:
 
 | 参数        | 说明           | 类型      | 默认值 | 版本 |
 | :---------- | :------------- | :-------- | :----- | :--- |
-| geometry    | 图形类型       | string    | 'line' | -    |
+| geometry    | 图形类型       | string    | line | -    |
```
Comment (dengfuping) [inline packages/charts/src/DualAxes/index.md:37]: - Suggest to highlight `enum` and `specific` value with `` like antd.

image

**Review by dengfuping — CHANGES_REQUESTED** [review]

File: packages/charts/src/DualAxes/index.md (hunk 2 above)
Comment (dengfuping) [reply packages/charts/src/DualAxes/index.md:37]: @wzc520pyfm Merge this PR first.

**Review by dengfuping — APPROVED** [review]

File: packages/charts/src/DualAxes/index.md (hunk 2 above)
Comment (wzc520pyfm) [reply packages/charts/src/DualAxes/index.md:37]: I will fix that in next PR.

**Pull request merged**